functests/sctp/sctp_sriov.go

```diff
@@ -0,0 +1,202 @@
+package sctp
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+
+	netattdefv1 "github.com/openshift/sriov-network-operator/pkg/apis/k8s/v1"
+	sriovv1 "github.com/openshift/sriov-network-operator/pkg/apis/sriovnetwork/v1"
+	sriovtestclient "github.com/openshift/sriov-network-operator/test/util/client"
+	sriovcluster "github.com/openshift/sriov-network-operator/test/util/cluster"
+	sriovdiscovery "github.com/openshift/sriov-network-operator/test/util/discovery"
+	sriovnamespaces "github.com/openshift/sriov-network-operator/test/util/namespaces"
+	sriovnetwork "github.com/openshift/sriov-network-operator/test/util/network"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/client"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/discovery"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/execute"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/namespaces"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/sriov"
+)
+
+const (
+	sriovOperatorNamespace = "openshift-sriov-network-operator"
+	testNetwork            = "test-sctp-sriov-network"
+)
+
+var _ = Describe("[sriov] SCTP integration", func() {
```

Comment:
how we are going to be able to run this only if sriov and sctp exist in the cluster?

Reply:
So, I was a bit biased here. My idea is that sriov is the hardest to get, so that's why I am putting the explicit selector here.
If we are not focusing on sriov, this test will be skipped.
Finally, one would want to test sriov BUT NOT SCTP and in that case my idea was to document this particolar case.